test/libsolidity/syntaxTests/errors/use_without_parentheses.sol

```diff
@@ -0,0 +1,7 @@
+error E1();
+error E2();
+function f() pure {
+    revert(E1);
+}
+// ----
+// TypeError 4423: (55-57): Expected error or string.
```

Comment:
I think this will be a bit confusing to some less experienced users. If would be better if the message could suggest that you need an error **instance**. Or maybe outright that the parentheses are missing.

Reply:
Improved the error message.